Added Advance tools to engineering cyborg

JayBaez opened the pull request:
What is in this PR:
Added the cyborg ATMOS holofan into the techweb for researching. You can research and print the upgrade board for the engineering cyborg. Added a new techweb research node for advance tools. The advance tools are the experimental tools (jaws of life, hand drill, tricorder, NOT experimental welding tool) and the Bluespace RPED. 

Purpose of this PR:
The engineering cyborg already has tools for humans, as the Geiger counter or analyzer, the ATMOS is too OP to have at the start so it can be researched. 
The BSRPED and advance tools give the borg a useful upgrade as the round advances.

:cl:  
rscadd: Added advance tools/ BSRPED/ ATMOS holofan to engineering cyborg
/:cl:

JayBaez committed:
Update robot_upgrades.dm

JayBaez committed:
Update all_nodes.dm

JayBaez committed:
Update all_nodes.dm

JayBaez committed:
Update all_nodes.dm

JayBaez committed:
Update layout.dm

JayBaez committed:
Update all_nodes.dm

JayBaez committed:
Update layout.dm

JayBaez committed:
Update all_nodes.dm

JayBaez committed:
Update mechfabricator_designs.dm

JayBaez committed:
Update robot_upgrades.dm

JayBaez committed:
Update mechfabricator_designs.dm

JayBaez committed:
Update robot_upgrades.dm

JayBaez committed:
ad

JayBaez committed:
tweaking mat cost and research stuff

JayBaez committed:
Update mechfabricator_designs.dm

JamieD1 reviewed (approved):
No experimental t ray

JamieD1 committed:
Apply suggestions from code review

JamieD1 reviewed (changes requested):
Remove the BSRPED from its own node and unlock it on the normal BSRPED to prevent wasting of resources if "noone will ever use it"

Also disable the option for disabling the barriers remotely.

Shout if you need help with any of these.

JamieD1 committed:
Update turdis.yml (#12323)

Yogbot-13 committed:
Automatic changelog generation #12323 [ci skip]

JamieD1 committed:
Delete AutoChangelog-pr-12323.yml

JamieD1 committed:
[BETA] - Lobby Screen (#12321)

* Replaced the lobby menu (with actual art) (#60953)

* Didn't mean to commit this

* This doesn't need changed anymore

* Dark mode

* Added countdown timer to lobby screen

* I hate conflicts

Co-authored-by: AMonkeyThatCodes <20987591+AMonkeyThatCodes@users.noreply.github.com>
Co-authored-by: Gabriel Adamson <adamson.g@gmail.com>

JamieD1 committed:
[BETA] Shuttle Building (#12322)

* Add files via upload

* changed sigtech, tourist, and clerk icons

added more detail to sigtech icon
new tourist and clerk icons

* Some more icon changes

I uploaded the wrong file

* applied grimm-hollowknight requested changes

Adds new signal technician and paramedic hud icons and reverts clerk's back, all requested by grimm-hollowknight

* ah, so thats why you reverted that change

im dumb

* why cant i merge upstream

why github

Co-authored-by: nmajask <nmajask@gmail.com>

JamieD1 committed:
Mission shitpossible (#12324)

* Infiltrators!

* Fix the cigarette vending machine in the infiltrator base

* Uncomment role preference

* Fix a bunch of bugs.

* Changes to AI hijack

* GODDAMIT I FORGOT TO SAVE IN STRONGDMM

* Fix the dumb gravgen runtime.

* Just throw the infiltrator base on centcom directly

* Add infiltrators to one-click antag (and fix vampires not being in it)

* blame jamie

* add infiltrators event

* Admin only for now.

Admin only for now.

* Maybe fix the round ending a BIT too early?

* Fix uplink not appearing properly in the roundend report

* Improve examine text for infiltrator power sink

* Whoops, remove the J

* More stuff.

* Jamie's map changes

* Make the AI hijacker more idiot-proof.

* Update map again.

* fix infiltrators not being able to use edaggers

* Update centcomm map again

* aaaaa

* fix powersink examine

* e

* wait nvm that's right

* big typo whoopsie

* Some big ol' improvements!

Co-authored-by: Lucy <lucy@absolucy.moe>

JamieD1 committed:
Custom shuttles (#12326)

* Ports "Ship construction through the power of science!"

- Ports https://github.com/BeeStation/BeeStation-Hornet/pull/1398

* Ports some changes

- Ports https://github.com/BeeStation/BeeStation-Hornet/pull/1904/files# and https://github.com/BeeStation/BeeStation-Hornet/pull/1529

* Should fix some stuff

- Removed some unused files
- Fixed a change to a random file

* Should fix more stuff

- Added some more important files
- Added the icons I forgot to add

* Oh, I didn't update the dme

- Should fix this, maybe?

* Probably should add the turf icons

- Added the turf icons

* Fixes the name filter

One down, two to go
- Fixes the name filter by using isnotpetty instead of OOC_FILTER_CHECK, now if you make a shuttle with a bad name, it will message the admins.

* Fixes the tech web

- Fixes the tech web
- Adds the new icons for the RSD

* Should fix some stuff with the shuttles

Should fix most of the runtimes and issues that ive had thanks to Jamie's help. Should bring this from ""functional"" to "functional".

* I am too tired for this

why github desktop, WHY!!

* Adds spaceship navigation beacons

They kinda work.

* should fix the beacons

why do i do this the way i do

* Ports some fixes by Dennok

Ports "Custom shuttle offsets and view range depends on ship size. #2685" by Dennok and "Custom transit dir for custom shuttle #2689" by Dennok

Co-authored-by: nmajask <nmajask@gmail.com>

JamieD1 committed:
Update layout.dm

JamieD1 committed:
Merge branch 'Beta' into jaybaez-patch-2

patpol4 reviewed (changes requested):
What is the point of the map change here? Looks like you're just doing changes from #12270 ?
